modules/node_modules/@ncigdc/containers/MostAffectedCasesTable.js

```diff
@@ -146,6 +167,183 @@ const MostAffectedCasesTableComponent = compose(
   );
 });
 
+const mutationsFragment10 = Relay.QL`
+fragment on Ssms {
+  row0: hits(filters: $ssmsFilters0) {
+    total
+  }
+  row1: hits(filters: $ssmsFilters1) {
+    total
+  }
+  row2: hits(filters: $ssmsFilters2) {
+    total
+  }
+  row3: hits(filters: $ssmsFilters3) {
+    total
+  }
+  row4: hits(filters: $ssmsFilters4) {
+    total
+  }
+  row5: hits(filters: $ssmsFilters5) {
+    total
+  }
+  row6: hits(filters: $ssmsFilters6) {
+    total
+  }
+  row7: hits(filters: $ssmsFilters7) {
+    total
+  }
+  row8: hits(filters: $ssmsFilters8) {
+    total
+  }
+  row9: hits(filters: $ssmsFilters9) {
+    total
+  }
+}
+`;
+
+const mutationsFragment20 = Relay.QL`
+fragment on Ssms {
+  row0: hits(filters: $ssmsFilters10) {
+    total
+  }
+  row1: hits(filters: $ssmsFilters11) {
+    total
+  }
+  row2: hits(filters: $ssmsFilters12) {
+    total
+  }
+  row3: hits(filters: $ssmsFilters13) {
+    total
+  }
+  row4: hits(filters: $ssmsFilters14) {
+    total
+  }
+  row5: hits(filters: $ssmsFilters15) {
+    total
+  }
+  row6: hits(filters: $ssmsFilters16) {
+    total
+  }
+  row7: hits(filters: $ssmsFilters17) {
+    total
+  }
+  row8: hits(filters: $ssmsFilters18) {
+    total
+  }
+  row9: hits(filters: $ssmsFilters19) {
+    total
+  }
+}
+`;
+
+const mutationsFragment40 = Relay.QL`
+fragment on Ssms {
+  row0: hits(filters: $ssmsFilters20) {
+    total
+  }
+  row1: hits(filters: $ssmsFilters21) {
+    total
+  }
+  row2: hits(filters: $ssmsFilters22) {
+    total
+  }
+  row3: hits(filters: $ssmsFilters23) {
+    total
+  }
+  row4: hits(filters: $ssmsFilters24) {
+    total
+  }
+  row5: hits(filters: $ssmsFilters25) {
+    total
+  }
+  row6: hits(filters: $ssmsFilters26) {
+    total
+  }
+  row7: hits(filters: $ssmsFilters27) {
+    total
+  }
+  row8: hits(filters: $ssmsFilters28) {
+    total
+  }
+  row9: hits(filters: $ssmsFilters29) {
+    total
+  }
+  row10: hits(filters: $ssmsFilters30) {
+    total
+  }
+  row11: hits(filters: $ssmsFilters31) {
+    total
+  }
+  row12: hits(filters: $ssmsFilters32) {
+    total
+  }
+  row13: hits(filters: $ssmsFilters33) {
+    total
+  }
+  row14: hits(filters: $ssmsFilters34) {
+    total
+  }
+  row15: hits(filters: $ssmsFilters35) {
+    total
+  }
+  row16: hits(filters: $ssmsFilters36) {
+    total
+  }
+  row17: hits(filters: $ssmsFilters37) {
+    total
+  }
+  row18: hits(filters: $ssmsFilters38) {
+    total
+  }
+  row19: hits(filters: $ssmsFilters39) {
+    total
+  }
+}
+`;
+
+const SsmVariables = {
```

Comment:
definitely not a good idea, but you could generate this list from relay
```
   mutationsFragment40.children.reduce((vars, row) => [...vars, ...row.calls.map(call => call.value.callVariableName)], [])
```
would give you an array of the variable names...
but don't do it haha probably pretty fragile.